### ISSUE TITLE
How to install Powerline fonts ?

### ISSUE THREAD

**Issue by Deathlive** [Neo-Oli/termux-ubuntu#20]
I installed [Oh-my-fish](https://github.com/oh-my-fish/oh-my-fish) and [bobthefish theme](https://github.com/oh-my-fish/theme-bobthefish) but some symbols not visible.
Tried `echo \ue0b0 \u00b1 \ue0a0 \u27a6 \u2718 \u26a1 \u2699`  instead of symbol unknown signs.
How to install this symbol ? I even installed the Termux style.

**Comment by Neo-Oli** [comment]
Most of the fonts in Termux:Styling are patched for Powerline. Try "DejaVu" for example.

**Comment by Deathlive** [comment]
@Neo-Oli maybe this is due to `command+=" TERM=$TERM"` ?

**Comment by Neo-Oli** [comment]
Yeah, I also just checked. Powerline fonts seem to work outside of ubuntu but not inside. It's probably an issue with LANG or TERM.

**Comment by Deathlive** [comment]
@Neo-Oli Maybe you found a solution? How to fix it ?

**Comment by Neo-Oli** [comment]
Try `echo -e "\ue0b0 \u00b1 \ue0a0 \u27a6 \u2718 \u26a1 \u2699"`. instead. That prints the correct symols for me.

**Comment by Deathlive** [comment]
![default](https://user-images.githubusercontent.com/29791715/32689751-384186ec-c703-11e7-84e4-bc36478ff2d1.png)

**Comment by Neo-Oli** [comment]
Can you paste the output of `env`? Especially TERM and LANG.

**Comment by Deathlive** [comment]
![default](https://user-images.githubusercontent.com/29791715/32690633-20577180-c713-11e7-881e-2be3c6fb8fa2.png)

**Comment by Deathlive** [comment]
@Neo-Oli Excuse me, maybe you found a solution for this problem?

**Comment by RupankarGhosh** [comment]
you need to run 
```
dpkg-reconfigure locales
```

**Comment by RupankarGhosh** [comment]
@Deathlive

Install powerline fonts.You don't need to install termux:styling app.there is a neat hack.install any powerline font and rename it to font.ttf and place it to ~/.termux/
